en/02-javascript/0202-modelo-de-datos/index.md

```diff
@@ -82,7 +82,7 @@ typeof {};
 typeof function () { return 42; };
 ```
 
-In JavaScript, it is possible to declare a variable without assigning a value to it. In this case, the variable's type would be `'undefined'`.

```

Comment:
This is a typographical choice by the author of the original Spanish text. I personally would prefer a less obtrusive form of emphasis, but I'll leave it as it is for the time being.